### PR TITLE
Fix auto annotation doc ##2843

### PR DESCRIPTION
**Pull request by waterfall414**
installation_automatic_annotation guide needs to update to reflect the path change

Change the mounting volume for nuctl command from  `pwd`/serverless/openvino/common to  `pwd`/serverless/common

### Motivation and context

nuctl complains it can't find python3 if I strictly follow the steps in guideline

### How has this been tested?

Tested in the fresh ubuntu 18.04 system

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by nmanovic** [comment]
@waterfall414 , thanks for the PR. Could you please take the latest develop branch and apply your changes? For now we have several conflicts in CHANGELOG.md

**Comment by waterfall414** [comment]
Hi @nmanovic , can you please review the updated PR? I am not experienced at the PR and hope I got it right this time.

File: CHANGELOG.md [hunk 1]
```diff
@@ -104,6 +105,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Extra shapes are drawn after Esc, or G pressed while drawing a region in grouping (<https://github.com/openvinotoolkit/cvat/pull/2507>)
 - Reset state (reviews, issues) after logout or changing a job (<https://github.com/openvinotoolkit/cvat/pull/2525>)
 - TypeError: Cannot read property 'id' of undefined when updating a task (<https://github.com/openvinotoolkit/cvat/pull/2544>)
+- Cannot find Python when deploying models through nuclio if steps are strictly followed in Installation_Automatic_Annotation.md
```
Comment (nmanovic) [inline CHANGELOG.md:108]: Need to remove one line. I can see the same statement several times in CHANGELOG.md

Comment (waterfall414) [reply CHANGELOG.md:112]: I just edited it in github web-client. Did it remove the duplicate?

File: CHANGELOG.md [hunk 2]
```diff
@@ -54,6 +54,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed cuboid perpective change (<https://github.com/openvinotoolkit/cvat/pull/2733>)
 - Annotation page popups (ai tools, drawing) reset state after detecting, tracking, drawing (<https://github.com/openvinotoolkit/cvat/pull/2780>)
 - Polygon editing using trailing point (<https://github.com/openvinotoolkit/cvat/pull/2808>)
+- Cannot find Python when deploying models through nuclio if steps are strictly followed in Installation_Automatic_Annotation.md (<https://github.com/openvinotoolkit/cvat/pull/2847>)
```
Comment (nmanovic) [inline CHANGELOG.md:57]:
```suggestion
- Updated the path to python for DL models inside automatic annotation documentation (<https://github.com/openvinotoolkit/cvat/pull/2847>)
```

**Review by nmanovic — APPROVED** [review]

**Pull request merged**